contracts/scripts/deploy.ts

```
@@ -0,0 +1,59 @@
+        ({ sierraCode, casmCode } = await getCompiledCode(
+        "peer_protocol_PeerProtocol"
+        ));
+    } catch (error: any) {
+        console.log("Failed to read contract files");
+        console.log(error);
+        process.exit(1);
+    }
+
+    const myCallData = new CallData(sierraCode.abi);
+    
+    const constructor = myCallData.compile("constructor", {
```

Biome

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)